qa/evil-tests/build.gradle

```diff
@@ -57,15 +57,6 @@ thirdPartyAudit {
   ignoreMissingClasses(
     'com.ibm.icu.lang.UCharacter'
   )
-
-  ignoreViolations(
```

Comment:
I honestly don't understand the difference here. It seems to be doing the right thing now where these exclusions are not needed. But it turns out the jimfs dependency is not needed here at all. I've opened #19347 to remove it from this project all together.